README.md

```diff
@@ -1,4 +1,4 @@
-# ALX SE Intranet Theme Selector - Chrome Extension
+![image](https://github.com/vinsky001/ALX-DarkMode-project/assets/84249407/c326edce-284e-4a20-a985-f81e76440b92)# ALX SE Intranet Theme Selector - Chrome Extension
```

Comment:
Perhaps you could get rid of the image. The other changes you're proposing would go a long way to helping our users find the extension easily on the store